main/templates/main/pages/blog.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'main/layouts/base-page.html' %}
+{% load static %}
+{% block title %}
+{% endblock %}
+{% block page-content %}
+<h1> <center> Representable Blog </center> </h1>
```

Comment:
the center tag is deprecated, would recommend styling the h1 instead https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center#:~:text=This%20tag%20has%20been%20deprecated,set%20margin%20to%200%20auto%20).